spec/services/candidate_submission_spec.rb

```diff
@@ -0,0 +1,87 @@
+require "rails_helper"
+
+RSpec.describe CandidateSubmission, :vcr do
+  let(:session) do
+    { registration: {
+      "first_name" => "joe",
+      "last_name" => "blogs",
+      "email" => "jo@example.com",
+      "subject_taught_id" => "6b793433-cd1f-e911-a979-000d3a20838a",
+      "degree_status_id" => "222750000",
+      "degree_type_id" => "222750000",
+      "degree_subject" => "6b793433-cd1f-e911-a979-000d3a20838a",
+      "uk_degree_grade_id" => "222750001",
+      "preferred_education_phase_id" => "222750000",
+      "has_gcse_maths_and_english_id" => "222750000",
+      "planning_to_retake_gcse_maths_and_english_id" => "222750000",
+      "has_gcse_science_id" => "222750000",
+      "planning_to_retake_gcse_science_id" => "222750000",
+      "initial_teacher_training_year_id" => "12917",
+      "date_of_birth" => "1980-01-01T00:00:00.000+00:00",
+      "country_id" => "00f5c2e6-74f9-e811-a97a-000d3a2760f2",
+      "address_line1" => "163",
+      "address_line2" => "hobbit hill",
+      "address_city" => "baggistown",
+      "address_postcode" => "SW1A 2ET",
+      "telephone" => "123456",
+      "accepted_policy_id" => ApiClient.get_latest_privacy_policy.id,
+      "preferred_teaching_subject_id" => "6b793433-cd1f-e911-a979-000d3a20838a",
+    } }
+  end
+  let(:equivalent_session) do
+    { registration: {
+      "first_name" => "joe",
+      "last_name" => "blogs",
+      "email" => "jo@example.com",
+      "degree_status_id" => "222750000",
+      "degree_type_id" => "222750005",
+      "preferred_teaching_subject_id" => "6b793433-cd1f-e911-a979-000d3a20838a",
+      "preferred_education_phase_id" => "222750000",
+      "initial_teacher_training_year_id" => "12917",
+      "date_of_birth" => "1980-01-01T00:00:00.000+00:00",
+      "country_id" => "00f5c2e6-74f9-e811-a97a-000d3a2760f2",
+      "address_line1" => "163",
+      "address_line2" => "hobbit hill",
+      "address_city" => "baggistown",
+      "address_postcode" => "SW1A 2ET",
+      "telephone" => "123456",
+      "accepted_policy_id" => ApiClient.get_latest_privacy_policy.id,
+      "phone_call_scheduled_at" => DateTime.now + 1.hour,
+    } }
+  end
+  let(:returner) { described_class.new(session, "accept_privacy_policy") }
+  let(:degree) { described_class.new(session, "degree/accept_privacy_policy") }
+  let(:studying) { described_class.new(session, "studying/accept_privacy_policy") }
+  let(:equivalent) { described_class.new(equivalent_session, "equivalent/accept_privacy_policy") }
+  let(:response_status) { YAML.load_file(VCR.current_cassette.file)["http_interactions"].last["response"]["status"]["code"] }
```

Comment:
I don't think we need to do this - it's just asserting that the static VCR file contains something, we probably just need to ensure that `whatever.call` doesn't raise an exception, which is what would happen if the request changes and no longer matches the VCR cassette:

```
expect(returner.call).to_not raise_error
```

Reply:
Good idea.